CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.2.0 - 2018-04-04
```

Comment:
Please put `## Unreleased` instead of fixed version and date for now. I'll have to update the contribution guide to state that this project follows https://keepachangelog.com

Reply:
Ok, I'll do it.